source/blog/2019-01-06-ovirt-openshift-part-1.html.md

```diff
@@ -0,0 +1,148 @@
+---
+title: oVirt and OKD
+author: rgolan
+tags: oVirt, oVirt 4.2, open source virtualization,kubernetes, openshift, external-storage, flexdriver, provisioner
+date: 2019-01-06 09:01:00 UTC
+---
+
+<img src="/images/blog/2019-01-06/boxhead.png" width="400px"/>
+
+
+
+This is a series of posts to demonstrate how to  install  OKD 3.11 on oVirt and what you can do with it.
```

Comment:
"This is the first blog, on a blog posts series focusing on Kubernetes deployment (OKD distribution) on top of oVirt, and an overview of the integration points between the projects"